<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedulerWidget</class>
 <widget class="QWidget" name="SchedulerWidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="jobsTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout">
     <item>
      <widget class="QLineEdit" name="cronEdit">
       <property name="placeholderText">
        <string>تعبير Cron</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>إضافة</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>حذف</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>تشغيل</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stopButton">
     <property name="text">
      <string>إيقاف</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
